PoGo.NecroBot.Logic/Tasks/FarmPokestopsGPXTask.cs

```diff
@@ -89,9 +89,15 @@ public static void Execute(Context ctx, StateMachine machine)
                         ctx.Navigation.HumanPathWalking(trackPoints.ElementAt(curTrkPt),
                             ctx.LogicSettings.WalkingSpeedInKilometerPerHour, () =>
                             {
+
                                 CatchNearbyPokemonsTask.Execute(ctx, machine);
```

Comment:
Just put UseNearbyPokestopsTask.Execute(ctx, machine); there, no need for a second function...